Configs/.local/share/hypr/lua/window_rules.lua

```
@@ -50,6 +50,7 @@ local floating =
       "Open File",
       "Volume Control",
       "Save As.*"
+      "File Already Exists — Dolphin"
```

Luacheck (1.2.0)

[error] expected '}' (to close '{' on line 36) near '"File Already Exists — Dolphin"'

(E011)